OpenRA.Mods.Common/Traits/Render/LeavesTrails.cs

```diff
@@ -20,10 +21,10 @@ public enum TrailType { Cell, CenterPosition }
 	[Desc("Renders a sprite effect when leaving a cell.")]
 	public class LeavesTrailsInfo : ConditionalTraitInfo
 	{
-		[FieldLoader.Require]
+		[Desc("Defaults to the actor name.")]
 		public readonly string Image = null;
 
-		[SequenceReference(nameof(Image))]
+		[SequenceReference(nameof(Image), allowNullImage = true)]
```

Comment:
This has to be `allowNullImage: true`?

Reply:
This does not make sense anymore since https://github.com/OpenRA/OpenRA/commit/b8e60ca8ec9a7076f44ed7c92ac43bc35a6f7a1e so I removed it.